functional/unit_action_test.go

```diff
@@ -73,48 +76,47 @@ func TestUnitSubmit(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	unitFile := "fixtures/units/hello.service"
+
 	// submit a unit and assert it shows up
-	if _, _, err := cluster.Fleetctl(m, "submit", "fixtures/units/hello.service"); err != nil {
+	_, _, err = cluster.Fleetctl(m, "submit", unitFile)
+	if err != nil {
 		t.Fatalf("Unable to submit fleet unit: %v", err)
 	}
-	stdout, _, err := cluster.Fleetctl(m, "list-units", "--no-legend")
```

Comment:
I just saw that one of the existing tests in this very file (namely `TestUnitRunnable()`) actually test both the count *and* the name -- so feel free to copy that if you really think it's worthwhile...